src/main/java/Duke.java

```diff
@@ -1,10 +1,92 @@
+import java.util.Scanner;
+
 public class Duke {
+
+     // Maximum number of tasks that can be held
+    private static final int MAX_TASK = 100;
+
+    // List of all tasks
+    private static Task[] Tasks = new Task[MAX_TASK];
+
+    // Total number of tasks in the list
+    private static int count = 0;
+
     public static void main(String[] args) {
+        displayWelcomeMessage();
+        while (true) {
+            String userCommand = getUserInput();
+            executeUserCommand(userCommand);
+        }
+    }
+
+    private static void displayWelcomeMessage() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(logo + "Hey there! I'm Duke.\n" + "How may I be of service?");
+    }
+
+    private static void displayGoodbyeMessage() {
+        System.out.println("Goodbye. Hope to see you again soon! :)");
+    }
+
+    private static String getUserInput() {
+        Scanner sc = new Scanner(System.in);
+        String userCommand = sc.nextLine();
+        return userCommand;
+    }
+
+    private static void executeUserCommand(String rawUserCommand) {
+        String userCommand = rawUserCommand;
+        int dividerPosition = rawUserCommand.indexOf(" ");
+        if (dividerPosition != -1) {
+            userCommand = rawUserCommand.substring(0, dividerPosition);
+        }
+        switch(userCommand) {
+        case "list":
+            displayList();
+            break;
+        case "done":
+            int taskNumber = Integer.parseInt(rawUserCommand.substring(dividerPosition+1));
```

Comment:
I would suggest to separate substring() and parseInt() into 2 different lines to improve readability.